### ISSUE TITLE
Errors in CSharp lexer file

### ISSUE THREAD

**Issue by Chris2011** [fredyvonvinet/ANTLR4-Plugins-for-NetBeans#39]
I created a ticket here: https://github.com/antlr/grammars-v4/issues/800

where I said, that I have one error in my grammar file, but as I commented, everything is fine in IntelliJ and Visual Studio Code. Now I don't know whether it is a general ANTLR problem with the lexer, that the lexer is wrong (I know the channel HIDDEN is missing) or is it a problem with your plugin?

Regards

Chris

**Comment by Chris2011** [comment]
As of this comment: https://github.com/antlr/grammars-v4/issues/800#issuecomment-315853071

the HIDDEN keyword should be a builtin so I think it is a bug in your plugin(?).

**Comment by Chris2011** [comment]
Still there after updating to 1.2.1.

**Comment by Chris2011** [comment]
No, why? Did you fix it? Please read the Ticket carefully and the linked Ticket. Thx.

**Comment by mcheung63** [comment]
i saw "Yes, this issue can be closed." in antlr/grammars-v4#800, can the issue close?

**Comment by Chris2011** [comment]
As I said, no it can't be closed, because the error still exists ^^. The comment was from an other repo, which has not much todo with this repo. This repo here brings a plugin for ANTLR into NetBeans with syntax hightlighting and generating. The other repo is the official antlr repo where they collect all the antlr grammar files as a repo.

I started to create the issue twice, here and there, because this plugin uses antlrv4 to generate the parser and lexer. It still shows error for the HIDDEN keyword so I decided to create the ticket at the antlr repo, because I thought it is a problem there. But in the ticket you can see comments that the hidden keyword is existing in general in antlr, so it must be an error in this plugin here.

**Comment by Chris2011** [comment]
There shouldn't be an error with HIDDEN.

**Comment by Chris2011** [comment]
The other ticket can be closed, because it seems not to be an error with the compiler of antlr.

**Comment by Chris2011** [comment]
Here I wrote:

As of this comment: antlr/grammars-v4#800 (comment)

the HIDDEN keyword should be a builtin so I think it is a bug in your plugin(?).
